app/src/main/kotlin/com/wire/android/media/audiomessage/AudioState.kt

```diff
@@ -3,10 +3,28 @@ package com.wire.android.media.audiomessage
 data class AudioState(
     val audioMediaPlayingState: AudioMediaPlayingState,
     val currentPositionInMs: Int,
-    val totalTimeInMs: Int
+    val totalTimeInMs: TotalTimeInMs
 ) {
     companion object {
-        val DEFAULT = AudioState(AudioMediaPlayingState.Stopped, 0, 0)
+        val DEFAULT = AudioState(AudioMediaPlayingState.Stopped, 0, TotalTimeInMs.NotKnown)
+    }
+
+    // before the user decides to play audio message, we are not able to determine total time ourself using
+    // MediaPlayer API, we are relaying on the info retrieved from the other client, until then
+    fun adjustTotalTime(otherClientTotalTime: Int): AudioState {
```

Comment:
Since this is only being invoked at the beginning to init the time and we are only checking if the total time is not known, can we rename to something more sensitive like `fun sanitizeTotalTime()` or `fun initTotalTime()`? Otherwise the function somehow implies that if the time will be used to set the total time also when `totalTimeInMs` is Known.